trinity/console.py

```diff
@@ -56,9 +60,16 @@ def console(ipc_path: Path,
     Method that starts the chain, setups the trinity CLI and register the
     cleanup function.
     """
+    logger = logging.getLogger("trinity")
     if env is None:
         env = {}
 
+    # if ipc_path is not found, raise an exception with a useful message
+    if not ipc_path.exists():
+        ipc_error_message = create_missing_ipc_error_message(ipc_path)
+        logger.error(ipc_error_message)
```

Comment:
My apologies for not being clear.  I do not think the logging message should be emitted within the `console` function, but instead, at the location where the exception is caught just before the `sys.exit(1)`.  You can use the exception message for the logging message.

```python
try:
    console(...)
except FileNotFoundError as err:
    logger.error(str(err))
    sys.exit(1)
```